src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -513,10 +513,32 @@ public Void caseTemplatizedType(TemplatizedType type) {
             emit("[]");
             return null;
           }
+          Iterator<JSType> it = type.getTemplateTypes().iterator();
+          if ("Object".equals(referencedType.getReferenceName())) {
+            emit("{ [");
```

Comment:
we have a test for that already in the `types.js`:
```
/** @type {Object} */
types.d = {};
```

Note this code is in `caseTemplatizedType`